Journalist independent missions (#5112)
endless-sky/endless-sky: data/human/transport missions.txt

			`	"It's more of a collage than an exact rendering," says the young woman you met in line. You were so distracted you didn't notice her approach. "They took 3D recordings from different moments and areas of Hope and edited them together in various loops.  The volcano was actually on the other side of the planet for example. The magic of editing. So what do you think?"`
			choice
				`	"Impressive."`
				`	"It's OK."`
			`	"It's just as I imagined it. Well, I have an exam to cram for. Enjoy it! I think it closes soon," she says as she heads for the exit.`
			`	You wander around the exhibit in awe. It's a bit disconcerting as the occasional hologram walks through you. Inside houses you see families frantically gathering their belongings and running for the Navy ships. It strikes you that somewhere here is a young Katya leaving her doomed home. You walk down one of the lines of refugees, observing each person. An officer slips in the snow and a young man helps him up.`

> 			`	As you move on you're startled to realize your face to face with the woman holding the baby, right where they were in the photo. She takes little steps toward the ship while bouncing the baby on her hip. A young woman with a camera and a respirator covering her face walks up to her, asks a question and begins to take photos. Before you can register whats going on the entire projection disappears in a disorienting flash, leaving you in a normal hanger bay. You are a little embarrassed to realize the exhibit is nearly empty as a woman walks up to you. "She loves that moment too. That's probably why she fought so hard to have that part left in, even though its a security risk."`

Reviewer: "You're face to face", "what's going on", "it's a security"

			`	"Captain," she says, her eyes lighting up. "Well, as you may know this is the last day of the expo. The captain I hired to transport everything back to <destination> is not only rude but now late. I would much rather have it in the hands of someone who appreciates the work. I was going to pay him 60000, but will pay you 80000. What do you say?"`
			choice
				`	"I would love to."`
				`	"Why not."`
				`	"Sorry, I have more pressing matters at hand."`
					goto oh
			`	"Great," she says. "Go and bring your ship to this hanger. I will have the crew get everything ready to load."`
				accept
			label oh
			`	"Oh, I'm surprised you would turn down so much money for such little work and a chance to meet the artists. But it's your call," she says. "The exhibit is closed. I'm going to have to ask you to leave."`
				decline
	on visit
		dialog `You arrive on <planet>, but realize that your escort carrying the exposition has not arrived yet. Better depart and wait for your escorts to enter the system.`
	on complete
		payment 80000
		log `Saw an impressive exposition on the Hope evaluation and ended up with a job taking the whole thing to Moonshake.`
		conversation

> 			`	You guide your ship down to the indicated hanger where there is a motley looking crew waiting to unload your ship. One of them hands you a note. It reads: "Meet me at the Common Grounds cafe in the spaceport if your interested in doing a project with us. Make sure your not followed. -Adelita Zero"`

Reviewer: "you're interested", "you're not followed".

mission "Journalism 2"
	description "Participate in the guerrilla filming of a Syndicate sweatshop on <destination>."
	name "Film Sweatshop"
	source "Moonshake"
	destination "Maker"
	passengers 4
	cargo "stealth drones" 1
	to offer
		has "Journalism 1: done"

	on offer
		conversation

> 			`You find the Common Grounds cafe. It has a funky vibe and it's quite busy. You don't seem to be recognized my anyone so you take a seat at a booth and order something to eat. Before your meal arrives a woman, maybe in her late thirties with dark hair, sits down across from you and hails the server. She cheerfully orders a smoothie with a long list of ingredients, many of which you have never heard of, or can even pronounce.`

Reviewer: "by anyone"

			`	"Sorry for the cloak and dagger bit," she says to you finally. "We have enemies in high places. So, Captain <last> "What did you think of the exposition?"`
			choice
				`	"I found it quite moving."`
				`	"It was OK."`
					goto wasok
				`	"It was too political for me."`
					goto political
			`	"I'm glad you liked it," she says smiling.`
				goto inorout
			label wasok

> 			`	"Not exactly a thrilling review, but your entitled to your opinion," she says smiling.`

Reviewer: "you're entitled"

				goto inorout
			label political
			`	"Oh. Well unfortunately for many, everything is political. It was political when the Republic tried to block the evacuation. And it was political when the Navy did it without permission. And the same goes for every single act of oppression and liberation since the dawn of humanity. Our oppressors are very political. Maybe you have the hope of being one some day; wealthy, authoritative. Or maybe you just don't care. Either way, you don't sound like a good fit for this project. But thanks for transporting the installation," she says with a smile. She gets her smoothie to-go and walks out.`
				decline
			label inorout

> 			`	Mariel said you saw me take the photos of the woman and her baby," you are surprised by the bluntness of her admission. "I really like that moment. Not many notice it. Your observant. Hopefully your kind too. We need a ship for our next project. We need you to transport a team to one planet, do a quick flyby of another, pickup the team and return here with everyone safe and sound. Nothing to dicey. But it's a dangerous universe, so who knows. Oh, and we have to leave today. I can't tell you more until you accept. It will pay, of course. 200000 credits. So what do ya say? Are you in or out?" Your orders are served and you eat in silence and consider her proposition while she drinks her smoothie.`

Reviewer: "You're observant.", "you're kind"

			choice
				`	"In."`
				`	"Out."`
					goto imout

> 			`	"Great. My name is Alejandra, but call me Ale. Here's the plan: you're going to drop me and two others off at Maker where we are going to film in 3D the conditions in a secrete Syndicate sweatshop. Thing is these aren't just poorly paid factory workers, but slaves. As you can imagine we can't get in as regular workers so with a bit of hacking we got ourselves jobs on site as private security. We know the Syndicate is getting the slaves from the pirates on Covert, and we also know where they do the transaction. So while we are getting the footage we need in the factories you and Bunker will spy on the Syndicate's rendezvous with the pirates on Hippocrates."`

Reviewer: "secret Syndicate"

			`	The line seems to be moving pretty quickly.`
			choice
				`	"Sure, why not."`
				` 	"Maybe some other time."`
					defer
			`	"You're going to love it," she says. "No one knows who they really are but Adelita's work is what inspired me to study journalism. Their mixture of mediums, their eye for story telling... I just love it. I have seen a lot of their stuff but never this expo on Hope. I'm so excited!"`
			`	You reach the entrance to the "expo" soon enough. Inside is a large room filled with photos, videos, interviews and other mixed media. They focus on three groups: the refugees of Hope, the Navy that saved them, and the Paradise planets that protested the evacuation. There is an installation with nearly a hundred original protest signs from rallies against the Dirt Belt and Navy. Some of them are quite crude and tasteless. Since you are on a Paradise planet it's a strong rebuke.`

> 			` There are many, many photos but one especially catches your eye. It's of a woman standing in line to board a Navy destroyer. She is holding a baby at her hip and staring at the camera with a look of sad determination. Their cheeks, rosy and chapped, stand out in contrast with the cold blues in the background. It gives you goosebumps.`

Reviewer 2: ```suggestion
			` There are many, many photos but one especially catches your eye. It's of a woman standing in line to board a Navy Frigate. She is holding a baby at her hip and staring at the camera with a look of sad determination. Their cheeks, rosy and chapped, stand out in contrast with the cold blues in the background. It gives you goosebumps.`
```

Author: went with cruisers.

> 			`	While good, you were hoping for a bit more as you head for the exit. As the automatic doors open you are hit with an unexpected arctic blast. Instead of back in the spaceport you find yourself in a life sized 3D recording of the evacuation of Hope, complete with refrigeration. The projection gives the illusion of a horizon. There are huge Navy destroyers slowly collecting ash in a snowy field near a small town as hundreds of refugees slowly board. But what stands out most is the giant volcano erupting in the background.`

Reviewer 2: ```suggestion
			`	While good, you were hoping for a bit more as you head for the exit. As the automatic doors open you are hit with an unexpected arctic blast. Instead of back in the spaceport you find yourself in a life sized 3D recording of the evacuation of Hope, complete with refrigeration. The projection gives the illusion of a horizon. There are huge Navy Cruisers slowly collecting ash in a snowy field near a small town as hundreds of refugees slowly board. But what stands out most is the giant volcano erupting in the background.`
```

> 			`	"It's more of a collage than an exact rendering," says the young woman you met in line. You were so distracted you didn't notice her approach. "They took 3D recordings from different moments and areas of Hope and edited them together in various loops.  The volcano was actually on the other side of the planet for example. The magic of editing. So what do you think?"`

Reviewer 2: ```suggestion
			`	"It's more of a collage than an exact rendering," says the young woman you met in line. You were so distracted by the experience you didn't notice her approach. "They took 3D recordings from different moments and areas of Hope and edited them together in various loops. The volcano was actually on the other side of the planet for example. The magic of editing. So what do you think?"`
```

			choice
				`	"Impressive."`
				`	"It's OK."`

> 			`	"It's just as I imagined it. Well, I have an exam to cram for. Enjoy it! I think it closes soon," she says as she heads for the exit.`

Reviewer 2: ```suggestion
			`	"It's just as I imagined it. Well, I have an exam to cram for. Enjoy the expo! I think it closes soon," she says as she heads for the exit.`
```

			`	As you move on you're startled to realize you're face to face with the woman holding the baby, right where they were in the photo. She takes little steps toward the ship while bouncing the baby on her hip. A young woman with a camera and a respirator covering her face walks up to her, asks a question and begins to take photos. Before you can register what's going on the entire projection disappears in a disorienting flash, leaving you in a normal hanger bay. You are a little embarrassed to realize the exhibit is nearly empty as a woman walks up to you. "She loves that moment too. That's probably why she fought so hard to have that part left in, even though it's a security risk."`
			`	"How rude of me," she says. "My name is Mariel, I organize these expositions and act as a spokesperson so the members of Adelita can remain anonymous. And you are?"`
			choice
				`	"Captain <last>."`

> 			`	"Captain," she says, her eyes lighting up. "Well, as you may know this is the last day of the expo. The captain I hired to transport everything back to <destination> is not only rude but now late. I would much rather have it in the hands of someone who appreciates the work. I was going to pay him 60000, but will pay you 80000. What do you say?"`

Reviewer 2: ```suggestion
			`	"Captain," she says, her eyes lighting up. "Well, as you may know this is the last day of the expo. The captain I hired to transport everything back to <destination> is not only rude but is now late. I would much rather have it in the hands of someone who appreciates the work. I was going to pay him 60000, but will pay you 80000. What do you say?"`
```

			`	"Sorry for the cloak and dagger bit," she says to you finally. "We have enemies in high places. So, Captain <last> "What did you think of the exposition?"`
			choice
				`	"I found it quite moving."`
				`	"It was OK."`
					goto wasok
				`	"It was too political for me."`
					goto political
			`	"I'm glad you liked it," she says smiling.`
				goto inorout
			label wasok
			`	"Not exactly a thrilling review, but you're entitled to your opinion," she says smiling.`
				goto inorout
			label political
			`	"Oh. Well unfortunately for many, everything is political. It was political when the Republic tried to block the evacuation. And it was political when the Navy did it without permission. And the same goes for every single act of oppression and liberation since the dawn of humanity. Our oppressors are very political. Maybe you have the hope of being one some day; wealthy, authoritative. Or maybe you just don't care. Either way, you don't sound like a good fit for this project. But thanks for transporting the installation," she says with a smile. She gets her smoothie to-go and walks out.`
				decline
			label inorout

> 			`	Mariel said you saw me take the photos of the woman and her baby," you are surprised by the bluntness of her admission. "I really like that moment. Not many notice it. You're observant. Hopefully you're kind too. We need a ship for our next project. We need you to transport a team to one planet, do a quick flyby of another, pickup the team and return here with everyone safe and sound. Nothing to dicey. But it's a dangerous universe, so who knows. Oh, and we have to leave today. I can't tell you more until you accept. It will pay, of course. 200000 credits. So what do ya say? Are you in or out?" Your orders are served and you eat in silence and consider her proposition while she drinks her smoothie.`

Author: I changed it around. I hope it's better.

			choice
				`	"In."`
				`	"Out."`
					goto imout

> 			`	"Great. My name is Alejandra, but call me Ale. Here's the plan: you're going to drop me and two others off at Maker where we are going to film in 3D the conditions in a secret Syndicate sweatshop. Thing is these aren't just poorly paid factory workers, but slaves. As you can imagine we can't get in as regular workers so with a bit of hacking we got ourselves jobs on site as private security. We know the Syndicate is getting the slaves from the pirates on Covert, and we also know where they do the transaction. So while we are getting the footage we need in the factories you and Bunker will spy on the Syndicate's rendezvous with the pirates on Hippocrates."`

Reviewer 2: ```suggestion
			`	"Great. My name is Alejandra, but call me Ale. Here's the plan: you're going to drop me and two others off at Maker where we are going to film in 3D the conditions in a secret Syndicate sweatshop. Thing is these aren't just poorly paid factory workers, but slaves. As you can imagine we can't get in as regular workers so with a bit of hacking we got ourselves jobs on site as private security. We know the Syndicate is getting the slaves from the pirates on Covert, and we also know where they do the transaction. So while we are getting the footage we need in the factories you and my partner Bunker will spy on the Syndicate's rendezvous with the pirates on Hippocrates."`
```

Author: I will implement this

> 			`	"Great. My name is Alejandra, but call me Ale. Here's the plan: you're going to drop me and two others off at Maker where we are going to film in 3D the conditions in a secret Syndicate sweatshop. Thing is these aren't just poorly paid factory workers, but slaves. As you can imagine we can't get in as regular workers so with a bit of hacking we got ourselves jobs on site as private security. We know the Syndicate is getting the slaves from the pirates on Covert, and we also know where they do the transaction. So while we are getting the footage we need in the factories you and Bunker will spy on the Syndicate's rendezvous with the pirates on Hippocrates."`

Author: "So while we are getting the footage we need in the factories you will be going with our associate Bunker to spy on the Syndicate's rendezvous with the pirates on Hippocrates"
does that help? Everything is kinda a secret up until that point so it's hard to mention him before.

> 			`	"Great. My name is Alejandra, but call me Ale. Here's the plan: you're going to drop me and two others off at Maker where we are going to film in 3D the conditions in a secret Syndicate sweatshop. Thing is these aren't just poorly paid factory workers, but slaves. As you can imagine we can't get in as regular workers so with a bit of hacking we got ourselves jobs on site as private security. We know the Syndicate is getting the slaves from the pirates on Covert, and we also know where they do the transaction. So while we are getting the footage we need in the factories you and Bunker will spy on the Syndicate's rendezvous with the pirates on Hippocrates."`

Author: I changed it around. Let me know if it's any better.

			`    "Don't worry," she reassures you, sensing your concern about spying on pirates and the Syndicate. "Bunker is a pro and we have an array of long range stealth drones that do the spying, you won't be anywhere near them. But pirates do frequent the system so be quick."`
			`	See waves to the server for the bill as she finishes the last gulp of her smoothie. "Let's go. Everyone will be meeting at the hanger shortly."`

> 			`	You go back to your ships hanger and sure enough there are three people with backpacks waiting for you with a few cases. They introduce themselves as Coya, River and Bunker.`

Reviewer 2: ```suggestion
			`	You and Ale go back to your ship's hanger and sure enough there are three people with backpacks waiting for you with a few cases. They introduce themselves as Coya, River and Bunker.`
```

	landing
	name "Slave trade"
	description "Do a flyby of <destination> while Bunker spies on the slave transaction."
	minor
	source "Maker"
	destination "Hippocrates"
	to offer
		has "Journalism 2: done"
	passengers 1
	cargo "stealth drones" 1
	on offer
		conversation
			`You drop off Ale, Coya and River at the spaceport and wish them luck. They are visibly embarrassed to be in security uniforms or perhaps uncomfortable with their disguises which make them virtually unrecognizable. Bunker can't stop laughing. Everyone exchanges wishes of good luck and they walk off into the spaceport. You and bunker go back to the <ship> and prepare to launch.`
				accept

	on visit
		dialog `You realize that your escort carrying the drones hasn't entered the system yet. Better depart and wait for it.`

####################################################################################################################

mission "Journalism 4"
	infiltrating
	landing
	name "Slave trade"
	description "Pick up the team on <destination>."
	minor
	source "Hippocrates"
	destination "Maker"
	to offer
		has "Journalism 3: done"
	passengers 1
	cargo "stealth drones" 1

	on offer
		conversation

> 			`It's not hard to find a remote canyon within range of the target area. You stay hovering in case you need a quick escape but are well inside the atmosphere so Bunker releases the drones from the open cargo bay without a pressure suit. They are so small you can't make them out from the cockpit, nor can you see them on your ships scanners. When Bunker returns he sets up a mini-3D display and sits at the controller. "OK, they are on auto-pilot to the rendezvous site."`

Reviewer 2: ```suggestion
			`It's not hard to find a remote canyon within range of the target area. You keep the <ship> hovering in case you need to make a quick escape, but as you are well within the atmosphere Bunker releases the drones from the open cargo bay without needing a pressure suit. The drones are so small you can't make them out from the cockpit, nor can you see them on your ship's scanners. When Bunker returns to the cockpit, he sets up a mini-3D display and sits at the controller. "OK, they are on auto-pilot to the rendezvous site."`
```

Reviewer 3: I'd recommend changing `the <ship>` to just `your ship`. Trying to anticipate what players will name ships and making that fit in the writing without causing grammatical problem is tough.

			`	Once the drones are on site you can see a lone syndicate ship with a security detail posted in front of the loading ramp. "The pirates must be late," says Bunker, smirking. "Typical. I'm going to scatter the drones and set a proximity alert."`
			`	He hums to himself for awhile before making small talk, "You know, there's a part of me that would much rather be liberating slaves instead of filming them. But for now it seems like this is the best way for me to help. Say we did liberate them right now. There is just going to be another slave ship tomorrow. But if we create political and social will; that could stop the slave ships forever."`
			choice
				`	"I think you're right."`
				`	"What about doing both?"`
					goto plan
			`	"And you know they will use those poor people as human shields. Say we..." he is cut off by the proximity alert.`
					goto showtime
			label plan
			`	"Now you're talking! But it would take a lot of planing to do it without killing the people we are trying to liberate. You know they will use those poor people as human shields. Say we..." he is cut off by the proximity alert.`
			label showtime

> 			`	"Show time," he says rubbing his hands together. You both watch as a pirate capital ship lands near the security detail, probably deafening them. It's accompanied by a small fleet of small escorts. As soon as the loading ramp is down the security detail gets to work herding the slaves from one ship to another. You see them hit and shove anyone that moves to slowly for their liking. It's hard to watch.`

Reviewer 2: ```suggestion
			`	"Show time," he says rubbing his hands together. You both watch as a pirate Leviathan lands near the security detail, probably deafening them. It's accompanied by a small fleet of Quicksilver escorts. As soon as the loading ramp is down the security detail gets to work herding the slaves from one ship to another. You see them hit and shove anyone that moves to slowly for their liking. It's hard to watch.`
```

> 			`	"Show time," he says rubbing his hands together. You both watch as a pirate capital ship lands near the security detail, probably deafening them. It's accompanied by a small fleet of small escorts. As soon as the loading ramp is down the security detail gets to work herding the slaves from one ship to another. You see them hit and shove anyone that moves to slowly for their liking. It's hard to watch.`

Reviewer 2: or replace "small escorts" with "Quicksilver escorts".

			choice
				`	"That's an order!"`
				`	"OK. Call the drones back."`
					goto calldrones
			`	"You're the captain," he responds giving up. You continue to watch the display. The last of the slaves are loaded followed by the security team. The two men are last to get to their ships. You notice a slight hum from outside before getting raked with fire from one of the pirate escorts. You try pull out to space but another escort appears, cutting you off and laying down a stream of fire. Your engines are hit and your ship falls from the sky, exploding on impact.`
				die
			label seeus
			`	"I have no idea! Right now we have bigger things to worry about."`
			choice
				`	"Call back the drones immediately."`
					goto escape
				`	"I'm sorry but we have to leave the drones behind."`
					goto nocando
			label nocando
			`	Bunker hits the recall button for the drones anyways. "This display is a low resolution proxy," explains Bunker. "If we leave the drones we lose all the footage we need. It looks like the fighters aren't coming straight for us. We have time. The drones are less than two minutes out."`
			choice
				`	"We're leaving. Now!"`

> 				`	"Fine. Get to the cargo bay lock down the drones. We take off on your mark."`

Reviewer 2: ```suggestion
				`	"Fine. Get to the cargo bay and lock down the drones as soon as they land. We take off on your mark."`
```

					goto escape
			`	"I can't let you do that <first>," says Bunker stepping in between you and the controls. "The drones are almost here. Just let me go to the cargo bay to lock them down and we're out of here.`
			choice
				`	(Pull your blaster and charge him with mutiny.)`
					goto blaster
				`	(Let him go to the cargo bay and lock down the drones.)`
					goto escape
			label calldrones
			`	"Aye aye captain," he says as he hits a button. "Drone ETA, two minutes, fifteen seconds."`

> 			`	"Get to the cargo bay," you say. "Lock down the drones and get belted in. We will take off on your mark."`

Reviewer 2: ```suggestion
			`	"Get to the cargo bay," you exclaim. "Lock down the drones and get belted in. We will take off on your mark."`
```

			label escape
			`	"Less than a minute later you hear Bunker on comms, "Drones are locked down and ready to go."`
			`	The moment you pull out of the canyon pirate fighters appear on your scanners. You accelerate into space at full speed with the fighters in hot pursuit.`
				launch
			label blaster
			`	You pull your blaster and order him to stand down. "Don't do this," he says. You're looking for the handcuffs when you notice a hum coming from outside. Suddenly you are raked with fire from one of the pirate escorts. You drop your blaster and try to lift off into space but another another escort appears, cutting you off and laying down a stream of fire. Your engines are hit and your ship falls from the sky and explodes on impact.`
				explode

	npc
		personality heroic nemesis
		government "Pirate"
		ship "Quicksilver" "Don't Run"
		ship "Quicksilver" "Obey"
		ship "Quicksilver" "Work is Freedom"

> 		ship "Quicksilver" "Falange"

Reviewer 4: A google search told me that "Falange" is the name of a former spanish political party. The name doesn't seem to fit with the rest so maybe it should be changed.

Author: changed it

		ship "Arrow" "Got You"
		ship "Arrow" "Futile"

	on visit

> 		dialog `You will want your escorts for this. Better take off and wait for them to enter the system.`

Reviewer 2: ```suggestion
		dialog `Your escort carrying the spy drones was left behind. Better take off and wait for them to enter the system.`
```
Give the player a clue why the story isn't progressing.

				launch
	on visit
		dialog `You left the <cargo> behind. Better take off and wait for them to enter the system.`
	on complete
		payment 220000
		log `Helped the Adelita collective gather video evidence of Syndicate participation in the slave trade.`
		conversation
			`They seem glad to be home. "Ugh. We saw such horrible things", says Ale, "and now we have to watch it over and over again to edit it into something presentable." She hands you a stack of credits. "There is a bonus in there for you. We really couldn't have done it without you. We will let you know when it's done. And we will definitely need you for some projects in the future. We'll be in touch."`

######################################################################################################################
#First Freedom Uprising missions
####################################################################################################################

mission "Journalism 6"
	deadline 9
	source "Trinket"
	destination "Harmony"
	to offer
		has "Journalism 5: done"
		or
			has "FW Southern Recon 1B: done"
			has "FW Southern Recon 1B (Turret): done"
	deadline 9
	on offer
		conversation

> 			`After talking with Ferya you receive an encrypted message from Ale. 'Hello there. The exposition is finally finished! We decided to start the tour in the Free Worlds. You're invited of course.'`

Reviewer 4: ```suggestion
			`After talking with Freya you receive an encrypted message from Ale. 'Hello there. The exposition is finally finished! We decided to start the tour in the Free Worlds. You're invited of course.'`
```

			`	"We weren't sure you were going to make it on such short notice," says Bunker. "So should we assume we have your endorsement? Full disclosure, we can only pay you for transport costs. It would be a conflict of interest if we paid you for access. You understand."`
			choice
				`	"I just came to see you both and wish you luck. I don't want to risk my good standing with the Council by endorsing this."`
				`	"Of course! I won't let you go into the hell of war in anyone's ship but my own."`
					goto greatnews
			`	Ale's excitement turns to disappointment. "I understand. I guess."`
			`	You try to make more small talk but they are clearly disappointed by your decision. "Well, maybe we will see you on the battlefield," she says sadly as you part ways. "We'll be in touch."`
				decline
			label greatnews
			`	"That's great news," says Ale. "How should we proceed?"`

> 			`	You explain your current mission. "Ferya wouldn't be to happy to see me without those Dreadnoughts. Best to get them first. Let's go. We can get caught up on the trip."`

Reviewer 4: ```suggestion
			`	You explain your current mission. "Freya wouldn't be too happy to see me without those Dreadnoughts. Best to get them first. Let's go. We can get caught up on the trip."`
```

					goto getgoing
				`	"I would love to."`
			`	They give you a tour of the factory as equipment is moved in piece by piece. Part of the factory has been converted into a kitchen and social space. The smell of their cooking makes your belly rumble. "Thanks to you we finally have some means of production," says Larisa. "We are starting off with making components for Megaparcec. The unions there are especially desperate for new trade partners. That will give them more leverage and autonomy from the Syndicate. And that's good for us because we need all the allies we can get."`
			`	"Food's ready!" calls out Lalo. In short order a few volunteers serve food to nearly a hundred people who take their plates outside and eat while conversing in small groups. The meal was simple but delicious. Larisa talks about plans to mine asteroids in the uninhabited systems you just passed through. Ale tells of when she and Bunker met Beto, Larisa and Lalo, agitating at a Megaparcec union strike. Lalo is pressured into talking about his part in the recent slave freeing raid and Bunker retells the story of when you and he filmed the Syndicate and the slavers. `

			label getgoing

> 			`	Ale and Bunker tell you they will be staying on Freedom for awhile to work on the project. "Don't worry about us. We will find transport back home when we are done here. We will be sure to let you know if we need you for more work," says Bunker as he hands you fist full of credits. "Unfortunately the pirates control the side of town with the spaceport so you will have to bribe them and land again if you want to refuel or buy anything." You say goodbye to everyone, promising to visit again soon.`

Reviewer 4: ```suggestion
			`	Ale and Bunker tell you they will be staying on Freedom for awhile to work on the project. "Don't worry about us. We will find transport back home when we are done here. We will be sure to let you know if we need you for more work," says Bunker as he hands you a fist full of credits. "Unfortunately the pirates control the side of town with the spaceport so you will have to bribe them and land again if you want to refuel or buy anything." You say goodbye to everyone, promising to visit again soon.`
```

			`The Testament enters the planet's atmosphere at high speed and lands somewhat uncontrolled near a waste processing plant. The cargo hold is glowing red hot and some areas in the cargo hold appear to be on fire.`
			`	Veronica disembarks, walks over to you, and apologizes for the difficult trip. She explains that the cooling systems failed, causing the sushi calamity you experienced during the multiple legs of your journey.`
			`	You are paid <payment> and she asks you not to talk about this escort job to anybody. You leave the docks quickly in order to escape the horrible smell of overheated fish that is coming from Veronica and her ship.`

	name "Take family home to <planet>"

> 	description "You meet a family worried about the potential destruction of war in the Free Worlds and want to cut their vacation short and go home to <destination>."

Reviewer 5: description "A family worried about the potential destruction in the Free Worlds wants to cut their vacation short and go home to <destination>."

	source
		government "Free Worlds"
		attributes "tourism"
	destination "Mainsail"
	passengers 3
	cargo "personal belongings" 1
	to offer
		has "FW Katya 7C: done"
	to fail
		has "free worlds plot completed"
	on offer
		conversation

> 			`After walking around the spaceport for awhile you head back to your ship to find and middle aged couple with their two grade school daughters waiting for you at your hanger. The man is busy trying to adjudicate a dispute between the girls but the lady notices you immediately and begins to approach you. "Thank goodness you're here," she says. "We were hoping you could get us back home to the paradise planets. This war caught us right in the middle of our vacation and we can't risk being anywhere near here when the Navy attacks, especially with our little 'angels'," she gestures toward her daughters, now in a full blown fight. "If you can find the time to take us we will pay you very well."`

Reviewer 5: This paragraph assumes the player has met this family before and also assumes that you know that the two girls are the couple's daughter(On this last point, assuming you can make it obvious, then it'd work.)

Author: ok. i will throw in a "you assume" somewhere in there.

			choice
				`	"Sure, I'll take you."`
				`	"OK. But it's pretty far. It better be worth it."`
				`	"No. I don't think so. Sorry."`
					decline

> 			`	"Then it's a deal," she shakes your hand firmly. The whole family gets onto the <ship> without another word, leaving you to load their many bags and cases.`

Reviewer 5: `	"Then it's a deal," she shakes your hand firmly. The whole family gets onto your ship without a word, leaving you to load their many bags and cases.`

				accept

	on complete
		payment 66999
		conversation

> 			`You're not sure if you have ever met a more pretentious and entitled family. The girls fought non-stop and both they all complained constantly of the lack of luxury accommodation on your ship. You are very happy to finally be free of them. Although you feel you should have been paid more for how terrible they were.`

Reviewer 5: `You're not sure if you have ever met a more pretentious and entitled family. The girls fought constantly and the family always complained about the lack of luxury accommodation on your ship. You are very happy to finally be free of them. Although you feel you should have been paid more for how terrible they were.`

Reviewer 5: Consider a branch where you have the luxury accommodation outfit.

Author: but it would have to be two separate missions to pay more for the luxury accommodation, right. Or can the payment tab go in a conversation? Honestly this mission was done hastily just to get the player to the trigger for the rest of the missions.  I wasn't even sure if its needed or wanted. I'm definitely down to make it better if folks think it should stay.

Reviewer 6: Two mutually exclusive mission versions would be easier ( I can attest to having tried similar).
It never hurts to have a more general mission take you to a trigger though, otherwise many things never get found. Just be careful it's not too easy for it to show up.

	on visit
		dialog phrase "generic cargo and passenger on visit"

mission "Journalism 1"
	name "Transport exposition to <planet>"
	description `Transport expo equipment to <destination>.`
	minor
	source
		attributes "paradise"
		not attributes "mining"
		not attributes "textiles"
		not attributes "research"
		not attributes "factory"
	destination "Moonshake"
	cargo "exposition installations" 5
	to offer
		random < 80
		has "Expedition to Hope 2: done"
		has "FW Katya 4: done"

	on offer
		conversation

> 			`As you explore the luxurious spaceport you notice a large group of people around one of the larger hangers.`

Reviewer 5: Describe the situation more.

			choice
				`	(Check it out.)`
				`	(Not right now.)`
					defer

> 			`	You approach some of the people at the end of the line and ask what's going on. "It's the last day of Adelita's exposition on the evacuation of Hope," says an excited young woman. "It was their first work that really got them famous. The last week of their expositions are always free. This is a great chance to see it if you haven't already."`

Reviewer 5: A. Lot. Of. Periods. Dialogue needs revision.

Author: Im not sure if three periods in a paragraph of dialog is a lot, but im a fan of Vonnegut. I will get change on to a comma.

> 			`	The line seems to be moving pretty quickly.`

Reviewer 5: Remove this line.

Author: The line about the line moving quickly?

			choice
				`	"I just came to see you both and wish you luck. I don't want to risk my good standing with the Council by endorsing this."`
				`	"Of course! I won't let you go into the hell of war in anyone's ship but my own."`
					goto greatnews
			`	Ale's excitement turns to disappointment. "I understand. I guess."`
			`	You try to make more small talk but they are clearly disappointed by your decision. "Well, maybe we will see you on the battlefield," she says sadly as you part ways. "We'll be in touch."`
				decline
			label greatnews
			`	"That's great news," says Ale. "How should we proceed?"`
			`	You explain your current mission. "Ferya wouldn't be to happy to see me without those Dreadnoughts. Best to get them first. Let's go. We can get caught up on the trip."`
				accept

	to complete
		has "FW Southern Battle 1: done"
	on visit
		dialog `You must wait for your escorts to reach this system.`

mission "Journalism 8"
	name "War Journalism"
	description "Take Ale and Bunker into the battle for Dancer."
	source "Trinket"
	destination "Dancer"
	to offer
		has "Journalism 7: done"
	passengers 2
	on offer
		conversation

> 			`"One more thing, if I may," you ask before leaving. "I would like to meet a couple of journalists who want to get embedded with the fleet and cover the war. You call in Ale and Bunker, introducing them and vouching for their trustworthiness. They make it clear to Freya that this will not be a pro-Free Worlds propaganda piece nor will the Free Worlds have any input on how the footage is used. But they promise to be very responsible with the recordings and not release anything that will be of use to the Navy. "The Navy's scans will be much more detailed than anything we record," reiterates Ale.`

Author: It should read "I would like you to meet a couple of journalists....",
Your way makes it seem like they just met.

> 			`"One more thing, if I may," you ask before leaving. "I would like to meet a couple of journalists who want to get embedded with the fleet and cover the war. You call in Ale and Bunker, introducing them and vouching for their trustworthiness. They make it clear to Freya that this will not be a pro-Free Worlds propaganda piece nor will the Free Worlds have any input on how the footage is used. But they promise to be very responsible with the recordings and not release anything that will be of use to the Navy. "The Navy's scans will be much more detailed than anything we record," reiterates Ale.`

Reviewer 2: That way is fine. However, what's there now sounds like <first> <last> is asking for permission to go find some journalists.

			`	"Y'all are so sweet," says Bunker. "Let's go tonight after the expo is closed. <first> is coming, right?"`
			choice
				`	"Wouldn't miss it for the galaxy."`
			`	"It's in the spaceport," says River. "Meet us there in a three hours or so, and wear these," they hand you a wig and some glasses. "Ah, and before I forget. Here is your payment"`

mission "Journalism 10"
	name "Syndicate Slavery exposition"
	description "Take Ale, Bunker and River to <planet>."
	source "Bourne"
	destination "Skymoot"
	to offer
		has "Journalism 9: done"
	passengers 3
	on offer
		conversation
			`The realities of interstellar travel means that spaceports never close and nothing in them is based on the light cycle of the planet it is on. The spaceport is as busy as any other hour when you get to the exposition.`
			`	"There he is," says Ale, pointing. You can barely make them out in their disguises. "Let's go. It's only closed a short while for cleaning."`
			`	Mariel, the curator, is inside waiting for you. "The exhibit will be clear for a half an hour. I hope every thing is as you specified."`

> 			`	"I'm sure everything is fine," says Ale. As before the first room is filled with photos and video interviews that you can only here when close. It looks like the Freedom Fighters gave them a few dozen shackles that they took off of slaves. They all lay broken in a heap in the center of the room. Everyone walks right past this and toward the door which you assume leads to the 3D projection.`

Author: "be heard" not "by heard" right... and sorry you made these corrections and i didn't get to them. The hidden messages mess me up.

			`	"Wow, that's really intense," says Ale as everyone leaves. "We really deserve a vacation. Hey <first>, we have always wanted to go to Skymoot to see the dragons, do you think you could take us?"`
			choice
				`	"I would be happy to."`
					goto happyto
				`	"I should really be getting back to the war."`
			`	"It's on the way but I can see you are busy. We will be in touch," she says, smiling.`
				decline
			label happyto
			`	"Wonderful. We will get our stuff and meet you at the <ship>," she says, smiling.`
				accept
	on visit
		dialog `You must wait for your escorts to reach this system.`
	on complete
		conversation
			`As you bring the <ship> down to the spaceport you see a dragon flying nearby and point it out to your passengers who 'ooo' and 'ahh'. They can hardly contain their excitement as they leave the ship. Each one gives you a hug and wishes you good luck.`
			`	"We will definitely be in touch," says Ale as they leave.`

mission "Freedom Uprising 1"
	landing
	deadline 14
	source
		government "Syndicate"
	destination "Moonshake"
	to offer
		has "Smuggler's Den, Part 2: done"
		has "Journalism 5: done"
	on offer
		conversation

> 			`	Back on your ship you notice a new message from Ale: "Hey. We are nearly done with the Syndicate Slavery project but it will still be a while for the exposition is ready to tour. In the meantime we have another project, but as usual it's time sensitive. Meet up with us on Moonshake soon or we will leave without you. Same place as before."`

Reviewer 2: ```suggestion
			`	Back on your ship you notice a new message from Ale: "Hey. We are nearly done with the Syndicate Slavery project but it will still be a while before the exposition is ready to tour. In the meantime we have another project, but as usual it's time sensitive. Meet up with us on Moonshake soon or we will leave without you. Same place as before."`
```

				accept

mission "Freedom Uprising 2"
	name "Road to Freedom"
	landing
	description "Take Ale, Beto and Bunker to <destination>."
	deadline 14
	source "Moonshake"
	destination "Sunracer"
	to offer
		has "Freedom Uprising 1: done"
		not "main plot completed"
	passengers 3
	on offer
		conversation
			`You find Ale and Bunker in the 'Common Ground Cafe' chatting with a young man in his mid-twenties. You notice they have backpacks and some cases with them. "There you are," says Bunker. "I was just telling them how you handled those pirate slavers."`
			`	"For like the hundredth time," says Ale, rolling her eyes.`
			`	"That's because it was epic. Aside from the thought of that pirate sensing the drones. If that's even what happened. It still gives me the chills.`
			choice
				`	"I haven't slept well since that day."`
				`	"I think it was just a coincidence."`

> 			`	"Whatever it was, it was weird. Ah, remember when we talked about liberating slave instead of filming them. Well this is Beto," he says, now more quietly.`

Reviewer 2: ```suggestion
			`	"Whatever it was, it was weird. Ah, remember when we talked about liberating slaves instead of filming them. Well this is Beto," he says, now more quietly.`
```

			`	"It's a pleasure to meet you Captain <last>," says Beto in a thick accent. "We have heard you played no small part in the recent independence of the Man system. This is exactly what we want too. But unfortunately the Republic is not as flexible as the Free Worlds, they would never help us."`
			`	"You see Beto here works with a group on Freedom," says Bunker. "They want to liberate their planet from the pirates and return to the ways of their ancestors; a system of autonomous assemblies and unions cooperating in mutual aid. You can say we support their work and they support ours. They recently made quite the stir on Freedom with a daylight raid on a slave market."`
			choice
				`	"You attacked the pirates on their own planet?"`

> 			`	"It's not their planet," exclaims Beto. "And were forced into action. They kidnapped and were about to sell some of our comrades as retribution for our activism. It was our first act of armed struggle. Until this point we have mostly been focused on opening a collectively owned and managed factory to provide options other than piracy or wage slavery. We had just inaugurated the first section of the factory when the slavers attacked."`

Reviewer 2: ```suggestion
			`	"It's not their planet," exclaims Beto. "And we were forced into action. They kidnapped and were about to sell some of our comrades as retribution for our activism. It was our first act of armed struggle. Until this point we have mostly been focused on opening a collectively owned and managed factory to provide options other than piracy or wage slavery. We had just inaugurated the first section of the factory when the slavers attacked."`
```

			`	"You see, half of my home town has been controlled by pirates since before I was born. The anti-slavery assemblies of West Mesa have maintained a tenuous peace with the pro-slavery pirates occupying East Mesa, but our efforts are raising tensions faster then anticipated. Even though we managed to liberate not only our comrades but all the slaves for sale that day, we have been woefully unprepared for the pirate reaction. In the last weeks a number of our comrades have been disappeared. Many in the movement have gone underground. We need guns, ships, equipment and most of all allies if we are to stand a chance of achieving our goals."`
			`	"That's where we come in," says Ale. "`

			choice
				`	"OK. Let's hear it."`
					goto hearit
				`	"I don't like where this is going."`
			`	"Just hear us out," she says.`

			label hearit

> 			`	"While their daring raid has brought them the ire of the pirates it has also gained them support in some unlikely places. We want to amplify that by going up to Freedom to interview fighters and freed slaves for an exposition. Of course they will be more likely to participate in the project if we help them, which is something we want to do anyways. So what we need from you is to pick up some supplies from near here and take it all to Freedom, along with us. Oh, and there will almost definitely be combat involved," she says with an innocent smile. "I really shouldn't tell you more until you accept."`

Reviewer 2: ```suggestion
			`	"While their daring raid has brought them the ire of the pirates, it has also gained them support in some unlikely places. We want to amplify that by going up to Freedom to interview fighters and freed slaves for an exposition. Of course they will be more likely to participate in the project if we help them, which is something we want to do anyways. So what we need from you is to pick up some supplies from near here and take it all to Freedom, along with us. Oh, and there will almost definitely be combat involved," she says with an innocent smile. "I really shouldn't tell you more until you accept."`
```

			`	"They can't give us the equipment outright," chimes in Bunker while nursing a beer. "But the union has put in an order for exactly what is needed for the factory on Freedom. They also arranged for union guys to do the transport. Unfortunately their ship, the 'Union Strong', will be 'attacked' in transit. Luckily for them you will be there just in time to save them. And by save them I mean board their ship and offload the equipment. After that we go to <destination> to do some interviews and pick up the rest of what we need before going to Freedom. It's a rough system but we should be safe once we land so hail the spaceport and Beto should be able to get us permission to land without paying protection."`
			`	"We are going to go though a lot of empty space soon so my contacts will have a Ramscoop waiting for us on Stormhold, but also consider installing more Fuel Pods here because they won't be available along the way."`

			choice
				`	"Sounds like a plan. Let's do it!"`
					accept

> 				`	"I though you all were against piracy?"`

Reviewer 2: ```suggestion
				`	"I thought you all were against piracy?"`
```

			`	"And we should resist the temptation of hanging around Alnitak. There is a Navy base on Farpoint and even though they are Oathkeepers they still might scan us. Lucky for us I know a couple of systems with good solar wind and few pirates.  Well, we better get moving or we will get stuck in this garden forever."`
				accept

	on complete
		outfit "Laser Rifle" -100
		outfit "Fragmentation Grenades" -10
		payment 200000
		conversation
			`You land near the factory and a crew comes out to unload the weapons and equipment. As you step outside you see the escorts have joined the meager 'Freedom Fighters' fleet overhead, ready for any pirate attack. Two people run up filled with excitement and embrace not only Beto but Ale and Bunker too. They seem to already know each other. "Lalo, Larisa. This is <first>. We couldn't have done it without them," says Beto by means of introduction.`
			`	"Is that so?", says Larisa. "Thank you so much. You must stay for dinner!"`
			choice
				`	"I really ought to be going."`
					goto getgoing
				`	"I would love to."`

> 			`	They give you a tours of the factory as equipment is moved in piece by piece. Part of the factory has been converted into a kitchen and social space. The smell of their cooking makes your belly rumble. "Thanks to you we finally have some means of production," says Larisa. "We are starting off with making components for Megaparcec. The unions there are especially desperate for new trade partners. That will give them more leverage and autonomy from the Syndicate. And that's good for us because we need all the allies we can get."`

Reviewer 2: ```suggestion
			`	They give you a tour of the factory as equipment is moved in piece by piece. Part of the factory has been converted into a kitchen and social space. The smell of their cooking makes your belly rumble. "Thanks to you we finally have some means of production," says Larisa. "We are starting off with making components for Megaparcec. The unions there are especially desperate for new trade partners. That will give them more leverage and autonomy from the Syndicate. And that's good for us because we need all the allies we can get."`
```

			`	"Food's ready!" calls out Lalo. In short order a few volunteers serve food to nearly a hundred people who take their plates outside and eat while conversing in small groups. The meal was simple but delicious. Larisa talks about plans to mine asteroids in the uninhabited systems you just passed through. Ale tells of when she and Bunker met Beto, Larisa and Lalo, agitating at a Megaparcec union strike. Lalo is pressured into talking about his part in the recent slave freeing raid and Bunker retells the story of when you and he filmed the Syndicate and the slavers. `

			label getgoing

> 			`	Ale and Bunker tell you they will be staying on Freedom for awhile to work on the project. "Don't worry about us. We will find transport back home when we are done here. We will be sure to let you know if we need you for more work," says Bunker as he hands you fist full of credits. "Unfortunately the pirates control the side of town with the spaceport so you will have to bribe them and land again if you want to refuel or buy anything." You say goodbye to everyone, promising to visit again soon.`

Reviewer 2: ```suggestion
			`	Ale and Bunker tell you they will be staying on Freedom for awhile to work on the project. "Don't worry about us. We will find transport back home when we are done here. We will be sure to let you know if we need you for more work," says Bunker as he hands you a fist full of credits. "Unfortunately the pirates control the side of town with the spaceport so you will have to bribe them and land again if you want to refuel or buy anything." You say goodbye to everyone, promising to visit again soon.`
```

			`The Testament enters the planet's atmosphere at high speed and lands somewhat uncontrolled near a waste processing plant. The cargo hold is glowing red hot and some areas in the cargo hold appear to be on fire.`
			`	Veronica disembarks, walks over to you, and apologizes for the difficult trip. She explains that the cooling systems failed, causing the sushi calamity you experienced during the multiple legs of your journey.`
			`	You are paid <payment> and she asks you not to talk about this escort job to anybody. You leave the docks quickly in order to escape the horrible smell of overheated fish that is coming from Veronica and her ship.`

mission "Family back to Paradise - no luxury"
	name "Take family home to <planet>"
	description "A family worried about the potential destruction in the Free Worlds wants to cut their vacation short and go home to <destination>."
	source
		government "Free Worlds"
		attributes "tourism"
	destination "Mainsail"
	passengers 3
	cargo "personal belongings" 1
	to offer
		has "FW Katya 7C: done"
	to fail
		has "free worlds plot completed"
	on offer
		require "Luxury Accommodations" 0
		conversation

> 			`After walking around the spaceport for awhile you head back to your ship to find and middle aged couple with what appear to be their two grade school daughters waiting for you at your hanger. The man is busy trying to adjudicate a dispute between the girls but the lady notices you immediately and begins to approach you. "Thank goodness you're here," she says. "We have been going from ship to ship looking for transport back home to the paradise planets. This war caught us right in the middle of our vacation and we can't risk being anywhere near here when the Navy attacks, especially with our little 'angels'," she gestures toward her daughters, now in a full blown fight. "If you can find the time to take us we will pay you very well."`

Author: I am more then happy to work on/ rewrite this. It was hastily put together so a few friends could play test it without missing the the first Journalism mission. Without changing it entirely maybe a line from the mom like "We were visiting family here when the terrorist bombings happened. We stayed thinking a political solution was possible but its obvious now that will not happen any time soon"...

> 			`After walking around the spaceport for awhile you head back to your ship to find and middle aged couple with what appear to be their two grade school daughters waiting for you at your hanger. The man is busy trying to adjudicate a dispute between the girls but the lady notices you immediately and begins to approach you. "Thank goodness you're here," she says. "We have been going from ship to ship looking for transport back home to the paradise planets. This war caught us right in the middle of our vacation and we can't risk being anywhere near here when the Navy attacks, especially with our little 'angels'," she gestures toward her daughters, now in a full blown fight. "If you can find the time to take us we will pay you very well."`

Reviewer 3: That line does make more sense. Go for it.

			choice
				`	"Sure, I'll take you."`
				`	"OK. But it's pretty far. It better be worth it."`
				`	"No. I don't think so. Sorry."`
					decline

> 			`	"Then it's a deal," she shakes your hand firmly. The whole family gets onto the <ship> without a word, leaving you to load their many bags and cases.`

Author: Instead of confident I was going for so desperate that they grab their chance as quick as possible and with little thought. I see your point and will add some detail to make it more clear.

				accept

	on complete

> 		payment 66999

Author: Yeah, its intentional. I was going to add the <payment> to the on complete convo to enfasize how money grubbing they are.

> 		payment 66999

Reviewer 3: Sounds good, just make sure it gets into the dialogue. I'd recommend changing it to a different number as well; the joke seems pretty cheap (no offense intended).

			`	"It's more of a collage than an exact rendering," says the young woman you met in line. You were so distracted by the experience you didn't notice her approach. "They took 3D recordings from different moments and areas of Hope and edited them together in various loops. The volcano was actually on the other side of the planet for example. The magic of editing. So what do you think?"`
			choice
				`	"Impressive."`
				`	"It's OK."`
			`	"It's just as I imagined it. Well, I have an exam to cram for. Enjoy the expo! I think it closes soon," she says as she heads for the exit.`
			`	You wander around the exhibit in awe. It's a bit disconcerting as the occasional hologram walks through you. Inside houses you see families frantically gathering their belongings and running for the Navy ships. It strikes you that somewhere here is a young Katya leaving her doomed home. You walk down one of the lines of refugees, observing each person. An officer slips in the snow and a young man helps him up.`

> 			`	As you move on you're startled to realize that you're face to face with the woman holding the baby, right where they were in the photo. She takes little steps toward the ship while bouncing the baby on her hip. A young woman with a camera and a respirator covering her face walks up to her, asks a question and begins to take photos. Before you can register what's going on the entire projection disappears in a disorienting flash, leaving you in a normal hanger bay. You are a little embarrassed to realize the exhibit is nearly empty as a woman walks up to you. "She loves that moment too. That's probably why she fought so hard to have that part left in, even though it's a security risk."`

Reviewer 3: ```suggestion
			`	As you move on, you're startled to find yourself face to face with the woman holding the baby, right where they were in the photo. She takes little steps toward the ship while bouncing the baby on her hip. A young woman with a camera and a respirator covering her face walks up to her, asks a question, and begins to take photos. Before you can register what's going on, the entire projection disappears in a disorienting flash, leaving you in a normal hanger bay. You are a little embarrassed to realize the exhibit is nearly empty as a woman walks up to you. "She loves that moment too. That's probably why she fought so hard to have that part left in, even though it's a security risk."`
```

> 			`	"How rude of me," she says. "My name is Mariel, I organize these expositions and act as a spokesperson for Adelita. And you are?"`

Reviewer 3: ```suggestion
			`	"Oh, you probably have no idea who I am. How rude of me," she says. "My name is Mariel. I organize these expositions and act as a spokesperson for Adelita. And you are?"`
```

			choice
				`	"Captain <last>."`
			`	"Captain," she says, her eyes lighting up. "Well, as you may know this is the last day of the expo. The captain I hired to transport everything back to <destination> is not only rude but now late. I would much rather have it in the hands of someone who appreciates the work. I was going to pay him 60000, but will pay you 80000. What do you say?"`
			choice
				`	"I would love to."`

> 				`	"Why not."`

Reviewer 3: ```suggestion
				`	"Why not?"`
```

				`	"Sorry, I have more pressing matters at hand."`
					goto oh

> 			`	"Great," she says. "Go and bring your ship to this hanger. I will have the crew get everything ready to load."`

Reviewer 3: ```suggestion
			`	"Great," she says. "Go and bring your ship to this hanger. I'll have the crew get everything ready to load."`
```

				accept
			label oh
			`	"Oh, I'm surprised you would turn down so much money for such little work and a chance to meet the artists. But it's your call," she says. "The exhibit is closed. I'm going to have to ask you to leave."`
				decline
	on visit

> 		dialog `You arrive on <planet>, but realize that your escort carrying the exposition has not arrived yet. Better depart and wait for your escorts to enter the system.`

Reviewer 3: ```suggestion
		dialog `You arrive on <planet>, but realize that your escort carrying the exhibition has not arrived yet. Better depart and wait for your escorts to enter the system.`
```

	on complete
		payment 80000

> 		log `Saw an impressive exposition on the Hope evaluation and ended up with a job taking the whole thing to Moonshake.`

Reviewer 3: ```suggestion
		log `Saw an impressive exhibition on the Hope evacuation and ended up with a job taking the entire setup to Moonshake.`
```

		conversation

> 			`	You guide your ship down to the indicated hanger where there is a motley looking crew waiting to unload your ship. One of them hands you a note. It reads: "Meet me at the Common Grounds cafe in the spaceport if you're interested in doing a project with us. Make sure you're not followed. -Adelita Zero"`

Reviewer 3: ```suggestion
			`	You guide your ship down to the indicated hanger where there is a motley-looking crew waiting to unload your ship. One of them hands you a note. It reads: "Meet me at the Common Grounds cafe in the Moonshake spaceport if you're interested in doing a project with us. Make sure you're not followed. -Adelita Zero"`
```
You might want to consider adding some additional text at some point explaining why this semi-secretive organization is willing to trust a random captain.

mission "Journalism 2"
	description "Participate in the guerrilla filming of a Syndicate sweatshop on <destination>."
	name "Film Sweatshop"
	source "Moonshake"
	destination "Maker"
	passengers 3
	cargo "stealth drones" 1
	to offer
		has "Journalism 1: done"

	on offer
		conversation

> 			`You find the Common Grounds cafe. It has a funky vibe and it's quite busy. You don't seem to be recognized by anyone so you take a seat at a booth and order something to eat. Before your meal arrives a woman, maybe in her late thirties with dark hair, sits down across from you and hails the server. She cheerfully orders a smoothie with a long list of ingredients, many of which you have never heard of, or can even pronounce.`

Reviewer 3: ```suggestion
			`You find the Common Grounds cafe. It has a funky vibe and it's quite busy. You don't seem to be recognized by anyone so you take a seat at a booth and order something to eat. Before your meal arrives, , sits down across from you and hails the server. She cheerfully orders a smoothie with a long list of ingredients, many of which you have never heard of, or can even pronounce. She seems to be in her late thirties, and has dark hair.`
```

> 			`	"Sorry for the cloak and dagger bit," she says to you finally. "We have enemies in high places. So, Captain <last> "What did you think of the exposition?"`

Reviewer 3: ```suggestion
			`	"Sorry for the cloak and dagger bit," she says to you finally. "We have enemies in high places. So, Captain <last> "What did you think of the exhibition?"`
```

			choice
				`	"I found it quite moving."`

> 				`	"It was OK."`

Reviewer 3: ```suggestion
				`	"It was okay."`
```

					goto wasok
				`	"It was too political for me."`
					goto political
			`	"I'm glad you liked it," she says smiling.`
				goto inorout
			label wasok
			`	"Not exactly a thrilling review, but you're entitled to your opinion," she says smiling.`
				goto inorout
			label political

> 			`	"Oh. Well unfortunately for many, everything is political. It was political when the Republic tried to block the evacuation. And it was political when the Navy did it without permission. And the same goes for every single act of oppression and liberation since the dawn of humanity. Our oppressors are very political. Maybe you have the hope of being one some day; wealthy, authoritative. Or maybe you just don't care. Either way, you don't sound like a good fit for this project. But thanks for transporting the installation," she says with a smile. She gets her smoothie to-go and walks out.`

Reviewer 3: ```suggestion
			`	"Oh. Well, unfortunately for many, everything is political. It was political when the Republic tried to block the evacuation. It was political when the Navy did it without permission. And the same goes for every single act of oppression and liberation since the dawn of humanity. Our oppressors are very political. Maybe you have the hope of being one some day: wealthy, authoritative. Or maybe you just don't care. Either way, you don't sound like a good fit for this project. But thanks for transporting the installation," she says with a smile. She gets her smoothie to-go and walks out.`
```

				decline
			label inorout

> 			`	Mariel said you saw me take the photos of the woman and her baby," you are surprised by the bluntness of her admission. "I really like that moment; not many notice it. You're observant but are you kind? We need a ship for our next project but I can't tell you much until you accept. We need you to transport a team to one planet, quickly land on another, then pickup the team and return here with everyone safe and sound. Nothing to dicey. But it's a dangerous universe, so who knows. Oh, and we have to leave today. It will pay, of course. 200000 credits. So what do you say? Are you in or out?" Your orders are served and you eat in silence and consider her proposition while she drinks her smoothie.`

Reviewer 3: ```suggestion
			`	"Mariel said you saw me take the photos of the woman and her baby." You're surprised by the bluntness of her admission. "I really like that moment; not many notice it. You're definitely observant. We need a ship for our next project, but I can't tell you much until you accept. We need you to transport a team to one planet, quickly land on another, then pickup the team and return here with everyone safe and sound. Nothing too dicey. But it's a dangerous universe, so who knows? Oh, and we have to leave today. It will pay, of course. 200,000 credits. So, what do you say? Are you in or out?" Your orders are served, and you eat in silence and consider her proposition while she drinks her smoothie.`
```
A meal is a long time to go without answering. You might want to change up the last sentence.

			choice

> 				`	"In."`

Reviewer 3: ```suggestion
				`	"I'm in."`
```

> 				`	"Out."`

Reviewer 3: ```suggestion
				`	"I'm out."`
```

					goto imout

> 			`	"Great. My name is Alejandra, but call me Ale. Here's the plan: you're going to drop me and a friend off at Maker where we are going to film in 3D the conditions in a secret Syndicate sweatshop. Thing is these aren't just poorly paid factory workers, but slaves. As you can imagine we can't get in as regular workers so with a bit of hacking we got ourselves jobs on site as private security. We know the Syndicate is getting the slaves from the pirates on Covert, and we also know where they do the transaction. So while we are getting the footage we need in the factories you will be going with our associate Bunker to spy on the Syndicate's rendezvous with the pirates on Hippocrates."`

Reviewer 3: ```suggestion
			`	"Great. My name is Alejandra, but call me Ale. Here's the plan: you're going to drop me and a friend off at Maker, where we'll 3D-film the conditions in a secret Syndicate sweatshop. The thing is, these aren't just poorly paid factory workers, but slaves. As you might imagine, we can't get in as regular workers. Fortunately, and with a bit of hacking, we got ourselves jobs on-site as private security. We know the Syndicate is getting the slaves from the pirates on Covert, and we also know where they do their transaction. So, while we're getting the footage we need in the factories, you'll be going with our associate Bunker to spy on the Syndicate's rendezvous with the pirates on Hippocrates."`
```

> 			`    "Don't worry," she reassures you, sensing your concern about spying on pirates and the Syndicate. "Bunker is a pro and we have an array of long range stealth drones that do the spying, you won't be anywhere near them. But pirates do frequent the system so be quick."`

Reviewer 3: ```suggestion
			`    "Don't worry," she reassures you, sensing your concern about spying on pirates and the Syndicate. "Bunker is a pro, and we have an array of long range stealth drones that do the spying. You won't be anywhere near them. Pirates do frequent the system though, so be quick."`
```

> 			`	See waves to the server for the bill as she finishes the last gulp of her smoothie. "Let's go. Everyone will be meeting at the hanger shortly."`

Reviewer 3: ```suggestion
			`	She waves to the server for the bill as she finishes the last gulp of her smoothie. "Let's go. Everyone will be meeting at the hanger shortly."`
```

> 			`	You go back to your ships hanger and sure enough there are two people with backpacks waiting for you with a few cases. They introduce themselves as River and Bunker.`

Reviewer 3: ```suggestion
			`	You go back to your ships hanger. Sure enough, there are two people with backpacks waiting for you with a few cases. They introduce themselves as River and Bunker.`
```

			choice

> 				`	"I am Captain <first> <last>."`

Reviewer 3: ```suggestion
				`	"I'm Captain <first> <last>."`
```

				`	"I'm <first>."`
					goto loadup

> 			`	They all grimace at your formality.`

Reviewer 3: This bit doesn't really add any substance tp the story. You might want to consider taking it out.

			label loadup

> 			`	"Good. Now that that's done with let's load up," says Bunker. "Taking down the man. Wwhhoo! Love it."`

Reviewer 3: ```suggestion
			`	"Good. Now that that's done with, let's load up," says Bunker. "Taking down the man. Wooo! Love it."`
```

				accept
			label imout

> 			`	"I'm sorry to hear that," she says between gulps of her smoothie. "Well I suppose it just wasn't meant to be," she finishes her "drink" with lightning speed, gets up and leaves.`

Reviewer 3: ```suggestion
			`	"I'm sorry to hear that," she says between gulps of her smoothie. "Well, I suppose it just wasn't meant to be," she finishes her "drink" with lightning speed, gets up and leaves.`
```
How come there are quotation marks around drink? Am I just not picking up on some innuendo?

				decline
	on visit
		dialog `The filming equipment is aboard an escort. Better take off and wait for it to arrive.`

mission "Journalism 3"
	landing
	name "Slave trade"
	description "Do a flyby of <destination> while Bunker spies on the slave transaction."
	minor
	source "Maker"
	destination "Hippocrates"
	infiltrating
	to offer
		has "Journalism 2: done"
	passengers 1
	cargo "stealth drones" 1
	on offer
		conversation

> 			`You drop off Ale and River at the spaceport and wish them luck. They are visibly embarrassed to be in security uniforms or perhaps uncomfortable with their disguises which make them virtually unrecognizable. Bunker can't stop laughing. Everyone exchanges wishes of good luck and they walk off into the spaceport. You and bunker go back to the <ship> and prepare to launch.`

Reviewer 3: ```suggestion
			`You drop off Ale and River at the spaceport and wish them luck. They are visibly embarrassed to be in security uniforms, or perhaps uncomfortable with their disguises which make them virtually unrecognizable. Bunker can't stop laughing. Everyone exchanges wishes of good luck and they walk off into the spaceport. You and Bunker go back to your <ship> and prepare to launch.`
```

				accept

	on visit
		dialog `You realize that your escort carrying the drones hasn't entered the system yet. Better depart and wait for it.`

mission "Journalism 4"
	landing
	name "Slave trade"
	description "Pick up the team on <destination>."
	minor
	source "Hippocrates"
	destination "Maker"
	to offer
		has "Journalism 3: done"
	passengers 1
	cargo "stealth drones" 1

	on offer
		conversation
			`It's not hard to find a remote canyon within range of the target area. You stay hovering in case you need a quick escape but are well inside the atmosphere so Bunker releases the drones from the open cargo bay without a pressure suit. They are so small you can't make them out from the cockpit, nor can you see them on your ships scanners. When Bunker returns he sets up a mini-3D display and sits at the controller. "OK, they are on auto-pilot to the rendezvous site."`

> 			`	Once the drones are on site you can see a lone syndicate ship with a security detail posted in front of the loading ramp. "The pirates must be late," says Bunker, smirking. "Typical. I'm going to scatter the drones and set a proximity alert."`

Reviewer 3: ```suggestion
			`	Once the drones are on site, you can see a lone Syndicate ship with a security detail posted in front of the loading ramp. "The pirates must be late," says Bunker, smirking. "Typical. I'm going to scatter the drones and set a proximity alert."`
```

> 			`	He hums to himself for awhile before making small talk, "You know, there's a part of me that would much rather be liberating slaves instead of filming them. But for now it seems like this is the best way for me to help. Say we did liberate them right now. There is just going to be another slave ship tomorrow. But if we create political and social will; that could stop the slave ships forever."`

Reviewer 3: ```suggestion
			`	He hums to himself for awhile before making small talk. "You know, there's a part of me that would much rather be liberating slaves instead of filming them. But, for now, it seems like this is the best way for me to help. If we did liberate them right now, there would still be another slave ship tomorrow. If we could create political and social outcry, however, that could stop the slave ships forever."`
```

			choice
				`	"I think you're right."`
				`	"What about doing both?"`
					goto plan

> 			`	"And you know they will use those poor people as human shields. Say we..." he is cut off by the proximity alert.`

Reviewer 3: ```suggestion
			`	"And you know they'll use those poor people as human shields. Say we-" he is cut off by the proximity alert.`
```

					goto showtime
			label plan

> 			`	"Now you're talking! But it would take a lot of planing to do it without killing the people we are trying to liberate. You know they will use those poor people as human shields. Say we..." he is cut off by the proximity alert.`

Reviewer 3: ```suggestion
			`	"Now you're talking! But it would take a lot of planing to do it without killing the people we're trying to liberate. You know they'll use those poor people as human shields. Say we-" he is cut off by the proximity alert.`
```

			label showtime

> 			`	"Show time," he says rubbing his hands together. You both watch as a pirate capital ship lands near the security detail, probably deafening them. It's accompanied by a small fleet of Quicksilver escorts. As soon as the loading ramp is down the security detail gets to work herding the slaves from one ship to another. You see them hit and shove anyone that moves to slowly for their liking. It's hard to watch.`

Reviewer 3: ```suggestion
			`	"Show time," he says rubbing his hands together. You both watch as a pirate capital ship lands near the security detail, probably deafening them. It's accompanied by a small fleet of Quicksilver escorts. As soon as the loading ramp, is down the security detail gets to work herding the slaves from one ship to another. You see them hit and shove anyone that moves to slowly for their liking. It's hard to watch.`
```

> 			`	"Pigs," blurts out Bunker. He zooms in on two men talking a few meters away from everyone else. "What do we have here?" he says. One is in a suit, the other a black hooded robe. "Must be the leaders," he whispers as if they can hear him. Just then, the hooded man looks up to the sky, as if he sees something. He motions and his escorts fire up their engines and begin taking off.`

Reviewer 3: ```suggestion
			`	"Pigs," mutters Bunker. He zooms in on two men talking a few meters away from everyone else. "What do we have here?" One is in a suit, but the other wears a black hooded robe. "Must be the leaders," he whispers, as if they can hear him. Just then, the hooded man looks up to the sky, as if he sees something. He motions, and his escorts fire up their engines and begin taking off.`
```

			` "I don't like the look of this," says Bunker."`
			choice
				`	"I'm sure it has nothing to do with us. Keep filming."`
				`	"Call back the drones immediately."`
					goto calldrones

> 				`	"I'm sorry but we have to leave the drones behind."`

Reviewer 3: ```suggestion
				`	"I'm sorry, but I think we have to leave the drones behind."`
```

					goto nocando

> 				`	"Did he SEE us?"`

Reviewer 3: ```suggestion
				`	"Did he see us?"`
```

					goto seeus
			`	"I don't know. I think we oughta get going," says Bunker, looking worried.`
			choice
				`	"That's an order!"`

> 				`	"OK. Call the drones back."`

Reviewer 3: ```suggestion
				`	"Okay, call the drones back."`
```

					goto calldrones

> 			`	"You're the captain," he responds giving up. You continue to watch the display. The last of the slaves are loaded followed by the security team. The two men are last to get to their ships. You notice a slight hum from outside before getting raked with fire from one of the pirate escorts. You try pull out to space but another escort appears, cutting you off and laying down a stream of fire. Your engines are hit and your ship falls from the sky, exploding on impact.`

Reviewer 3: ```suggestion
			`	"You're the captain," he responds, giving up. You continue to watch the display. The last of the slaves are loaded, followed by the security team. The two men are last to get to their ships. You notice a slight hum from outside, than are suddenly raked with fire from one of the pirate escorts. You try to pull out to space, but another escort appears, cutting you off and laying down a stream of fire. Your engines are hit and your ship falls from the sky, exploding on impact.`
```

				explode
			label seeus
			`	"I have no idea! Right now we have bigger things to worry about."`
			choice
				`	"Call back the drones immediately."`
					goto calldrones

> 				`	"I'm sorry but we have to leave the drones behind."`

Reviewer 3: ```suggestion
				`	"I'm sorry, but I think we have to leave the drones behind."`
```

					goto nocando
			label nocando

> 			`	Bunker hits the recall button for the drones anyways. "This display is a low resolution proxy," explains Bunker. "If we leave the drones we lose all the footage we need. It looks like the fighters aren't coming straight for us. We have time. The drones are less than two minutes out."`

Reviewer 3: ```suggestion
			`	Bunker hits the recall button for the drones anyways. "This display is a low resolution proxy," explains Bunker. "If we leave the drones, we lose all the footage we need. It looks like the fighters aren't coming straight for us. We have time. The drones are less than two minutes out."`
```

			choice
				`	"We're leaving. Now!"`
				`	"Fine. Get to the cargo bay and lock down the drones as soon as they land. We take off on your mark."`
					goto escape
			`	"I can't let you do that <first>," says Bunker stepping in between you and the controls. "The drones are almost here. Just let me go to the cargo bay to lock them down and we're out of here.`
			choice
				`	(Pull your blaster and charge him with mutiny.)`
					goto blaster
				`	(Let him go to the cargo bay and lock down the drones.)`
					goto escape
			label calldrones

> 			`	"Aye aye captain," he says as he hits a button. "Drone ETA, two minutes, fifteen seconds."`

Reviewer 3: ```suggestion
			`	"Aye aye captain," he says as he hits a button. "Drone ETA: two minutes, fifteen seconds."`
```

> 			`	"Get to the cargo bay," you say. "Lock down the drones as soon as they arrive and get belted in. We will take off on your mark."`

Reviewer 3: ```suggestion
			`	"Get to the cargo bay," you say. "Lock down the drones as soon as they arrive and get belted in. We'll take off on your mark."`
```

			label escape

> 			`	Less than a minute later you hear Bunker on comms, "Drones are locked down and ready to go."`

Reviewer 3: ```suggestion
			`	Less than a minute later, you hear Bunker on comms. "Drones are locked down and ready to go."`
```

> 			`	The moment you pull out of the canyon pirate fighters appear on your scanners. You accelerate into space at full speed with the fighters in hot pursuit.`

Reviewer 3: ```suggestion
			`	The moment you pull out of the canyon, pirate fighters appear on your scanners. You accelerate into space at full speed with the fighters in hot pursuit.`
```

				launch
			label blaster

> 			`	You pull your blaster and order him to stand down. "Don't do this," he says. You're looking for the handcuffs when you notice a hum coming from outside. Suddenly you are raked with fire from one of the pirate escorts. You drop your blaster and try to lift off into space but another another escort appears, cutting you off and laying down a stream of fire. Your engines are hit and your ship falls from the sky and explodes on impact.`

Reviewer 3: ```suggestion
			`	You pull your blaster and order him to stand down. "Don't do this," he says. You're looking for the handcuffs when you notice a hum coming from outside. Suddenly, you are raked with fire from one of the pirate escorts. You drop your blaster and try to lift off into space but another escort appears, cutting you off and laying down a stream of fire. Your engines are hit and your ship falls from the sky, exploding on impact.`
```

				explode

	npc
		personality heroic nemesis vindictive launching
		government "Pirate"
		ship "Quicksilver" "Don't Run"
		ship "Quicksilver" "Obey"
		ship "Quicksilver" "Work Is Freedom"
		ship "Quicksilver" "Do As You're Told"
		ship "Quicksilver" "Got You"
		ship "Quicksilver" "Futile"

	on visit
		dialog `Your escort carrying the spy drones was left behind. Better take off and wait for them to enter the system.`

mission "Journalism 5"
	name "Slave trade"
	description "Bring everyone back to <destination>."
	minor
	source "Maker"
	destination "Moonshake"
	landing
	to offer
		has "Journalism 4: done"
	passengers 3
	cargo "stealth drones" 1

	on offer
		conversation
			branch experienced
				"combat rating" > 100

> 			`It was close but you make it back to Maker. Ale and River are excited to get on board and out of their uniforms and disguises. They are exhausted and eager to get home and process what they filmed, emotionally as well as literally. "That was horrible, but we got it done. Can we leave now?" asks Ale.`

Author: On the contrary your criticism is highly constructive. My goal is to create a good story the makes it into the game, so i greatly appreciate the time you have taken to help me make it better.

				launch
			label experienced
			`The pirates were no match for you. Ale and River are excited to get on board and out of their uniforms and disguises. They seem exhausted and eager to get home and process what they filmed, as well as emotionally. "That was awful. Can we leave now?" asks Ale.`
				launch
	on visit
		dialog `You left the <cargo> behind. Better take off and wait for them to enter the system.`
	on complete
		payment 220000

> 		log `Helped the Adelita collective gather video evidence of Syndicate participation in the slave trade.`

Author: I need to work on the logs of most of the missions. It took a back seat when i was writing. Now that a few people have looked it over and i feel a bit more confidant i will be adding them soon.

> 		log `Helped the Adelita collective gather video evidence of Syndicate participation in the slave trade.`

Reviewer 3: Good to hear!

		conversation
			`They seem glad to be home. "Ugh. We saw such horrible things", says Ale, "and now we have to watch it over and over again to edit it into something presentable." She hands you a stack of credits. "There is a bonus in there for you. We really couldn't have done it without you. We will let you know when it's done. And we will definitely need you for some projects in the future. We'll be in touch."`

mission "Journalism 6"
	deadline 9
	source "Trinket"
	destination "Harmony"
	to offer
		has "Journalism 5: done"
		or
			has "FW Southern Recon 1B: done"

> 			has "FW Southern Recon 1B (Turret): done"

Author: Im pretty sure if we do that we make the mission impossible to offer. Those missions are mutually exclusive. When you bring back the electron beam (either normal or turret, but not both like you put) the science team tells you Freya is waiting for you in a cafe. If more than one missing is offered at the same time they are ordered alphabetically. Since the F in "FW Southern Recon 1b" is before the J in "Journalism..." it should be offered right after you speak with Freya about going to get the Dreadnoughts.   Im pretty sure this worked well with i play tested. But i can check again.

> 			has "FW Southern Recon 1B (Turret): done"

Author: But as you mentioned before, this might be where its most cluttered with missions. It would be a bummer but i am not entirely opposed to removing most of the mission and just having an invitation to the opening of the expo. Another idea is instead of meeting up with them so they can go with you to the battle, you just meet up with one of their contacts in Zug who installs the cameras. Then after the battle you meet up with them for the expo. 
Again, i am happy to talk about this on discord as well but where? Im new to the platform. My username is joshu

> 			has "FW Southern Recon 1B (Turret): done"

Reviewer 3: I do like your second option to avoid taking the mission out all together. Yeah, content creating is probably the best spot, but "content creating" encapsulates so many different things that story is rarely talked about. Besides, most people creating new stories right now are  experienced so detached from the rest of established space that there's rarely a need for them to talk about what they're writing. That does make it tough. My user's Wizrad; feel free to shoot me a DM or ping me in content-creating. I do partly like the idea of extending the Free Worlds breaks, but that might be a hard sell to a lot of the core members who would be the ones that decide whether that's something looking into. Also, if you don't know, make sure you load the hidden conversations on github. I had quite a few more suggested changes that you haven't addressed yet and wanted to make sure you'd seen they were there since you seem to have resolved the majority of changes that weren't automatically collapsed.

> 			has "FW Southern Recon 1B (Turret): done"

Author: Im not sure what you mean by "load the hidden conversations". And looks like i need your four digit tag to find you on discord.

> 			has "FW Southern Recon 1B (Turret): done"

Reviewer 7: Number should be 3665. See attached image to see what I'm talking about. You want to click "Load more...".
![Screenshot (2)](https://user-images.githubusercontent.com/20505367/87447393-87298e00-c5c8-11ea-8999-7232704e2551.png)

			choice
				`	"Sure, I'll take you."`
				`	"Okay, but that's pretty far. This better be worth it."`
				`	"No, I don't think so. Sorry."`
					decline
			`	"Then it's a deal," she shakes your hand firmly. The whole family gets onto the <ship> without a word, leaving you to load their many bags and cases.`
				accept

	on complete
		payment 66999
		conversation
			`You're not sure if you have ever met a more pretentious and entitled family. The girls fought non-stop and they all complained constantly of the lack of luxury accommodation on your ship. The lady gives you <payment> and they leave without a word. They never even told you their names. You are glad to finally be free of them. Although you feel you should have been paid more for how terrible they were.`

> 	on visit

Reviewer 8: Mission structure should follow the style of existing missions. That is, on visit should go above on complete.

Author: Thanks for looking it over. I will work on the style changes this coming week. And my apologies. I somehow missed those suggestions in the hidden conversations. The I will implement the ones that were to old to commit. Im not sure exactly what you mean about eating the white space but i assume it has to do with when i switched to atom as my text/code editor, which messed with the spacing with I first used it. I thought I got it sorted. I will give it another look.  And finally, part of me had given up on this story, mostly because a couple people told me it was very unlikely to make it into the game. I assume since others are still making corrections there is still a chance? I had started writing a "neutral" / "guildish" storyline instead. Any advice for which of the two ideas is more productive for the game (and thus more likely to be included)?

				launch
			label experienced
			`The pirates were no match for you. Ale and River are soon on board and excited to be out of their disguises. Although exhausted, both are eager to get back and process what they filmed- both emotionally and literally. "That was awful. Can we leave now?" asks Ale.`
				launch
	on visit
		dialog `You left the <cargo> behind. Better take off and wait for them to enter the system.`
	on complete
		payment 220000
		log `Helped the Adelita collective gather video evidence of Syndicate participation in the slave trade.`
		conversation
			`They seem glad to be home. "Ugh. We saw such horrible things", says Ale, "and now we have to watch it over and over again to edit it into something presentable." She hands you a stack of credits. "There's a bonus in there for you. We really couldn't have done it without you. We'll let you know when it's done. Also, we might need you for some future projects, so expect us to be in touch." The group leaves your ship.`

mission "Journalism 6"
	deadline 9
	source "Trinket"
	destination "Harmony"
	to offer
		has "Journalism 5: done"
		or
			has "FW Southern Recon 1B: done"
			has "FW Southern Recon 1B (Turret): done"

> 	deadline 9

Reviewer 8: Style: deadline goes above the to offer.

	on complete
		conversation
			`As you land at the spaceport, you see a dragon flying nearby and point it out to your passengers. They can hardly contain their excitement as they leave the ship. Each one gives you a hug and wishes you good luck.`
			`	"We will definitely be in touch. Until next time!" says Ale as they leave.`

mission "Freedom Uprising 1"
	landing
	deadline 14
	source
		government "Syndicate"
	destination "Moonshake"
	to offer
		has "Smuggler's Den, Part 2: done"
		has "Journalism 5: done"
	on offer
		conversation
			`	Back on your ship you notice a new message from Ale: "Hey. We are nearly done with the Syndicate Slavery project but it will still be a while for the exposition is ready to tour. In the meantime we have another project, but as usual it's time sensitive. Meet up with us on Moonshake soon or we will leave without you. Same place as before."`
				accept

mission "Freedom Uprising 2"
	name "Road to Freedom"
	landing
	description "Take Ale, Beto and Bunker to <destination>."
	deadline 14
	source "Moonshake"
	destination "Sunracer"
	to offer
		has "Freedom Uprising 1: done"
		not "main plot completed"

> 	passengers 3

Reviewer 8: Style: passengers should go above the to offer.

			`	"I'm sure they will take their time," she says after a moment. "I'm going to explore for awhile. Let's meet up in the spaceport in a few hours."`
				accept

mission "Freedom Uprising 3"
	name "Road to Freedom"
	description "Take Ale, Beto and Bunker to <destination>. Make sure to have plenty of cargo space and fuel capacity."
	source "Sunracer"
	destination "Stormhold"
	cargo "Factory Equipment" 79
	blocked "You are going to need more cargo space for the coming mission."
	clearance `Beto manages to convince the pirate spaceport to let you land "free of charge."`
	waypoint "Zaurak"

> 	on enter "Zaurak"

Reviewer 8: Style: on enter should go after on offer and before on complete.

			`	"We are going to go though a lot of empty space soon so my contacts will have a Ramscoop waiting for us on Stormhold, but also consider installing more Fuel Pods here because they won't be available along the way."`

			choice
				`	"Sounds like a plan. Let's do it!"`
					accept
				`	"I thought you all were against piracy?"`
			`	"We aren't against piracy per se," says Beto. "There is nothing wrong about stealing from your oppressor. But many pirates are individualists who will rob, kill or enslave anyone for their own self gain. That we can not abide by."`

			choice
				`	"This is nuts. I can't steal from the Syndicate."`
					goto steal
				`	"OK. Let's do this."`
					accept
			label steal
			`	"I understand," says Ale. "It's a big risk and a lot to ask. We will be in touch if there is other more tame work available."`
				decline

> 	on visit dialog `You either forgot to board the 'Union Strong' or you left your escorts behind.`

Reviewer 8: You're missing a new line here.

	on complete
		payment 1000000

> 		outfit "Ramscoop" +1

Reviewer 8: You don't need a plus sign for `outfit`. Any number that isn't negative is already adding.
```suggestion
		outfit "Ramscoop" 1
```

		conversation
			`	A few people are waiting with a Ramscoop when you land. "So far so good," says Bunker. "Beto wants us to interview some allies and ex-slaves here on <planet>. While we do that can you go to the outfitter and buy 100 Laser Rifles and 10 boxes of Frag Grenades? Here's the credits, keep the change. And can you make sure the Ramscoop is installed? Meet us at 'The Giving Tree' when we're done."`

> 	npc assist

Reviewer 8: Style: npc blocks go after on offer and above on complete.
